client/src/Login/Login.js

```diff
@@ -52,21 +53,23 @@ export function Login() {
     })
     if (!response.ok || response.headers.get('authorization') === null) {
       const errorMessage = await response.json()
+      dispatch(removeAuth())
```

Comment:
Do you mind explaining why we need this?

Reply:
We probably don't.  I think I just put it in there as a proactive paranoid measure, clear out any stored data if they get a failed login attempt.  I'm okay with removing it.